homeassistant/components/matrix/manifest.json

```diff
@@ -2,6 +2,11 @@
   "domain": "matrix",
   "name": "Matrix",
   "documentation": "https://www.home-assistant.io/integrations/matrix",
-  "requirements": ["matrix-client==0.3.2"],
-  "codeowners": ["@tinloaf"]
+  "requirements": [
+    "matrix-client==0.3.2"
+  ],
+  "codeowners": [
+    "@tinloaf"
```

Comment:
Since you're making significant changes, can you add yourself so you are notified when new issues related to this come in.